server/routes/categories.js

```diff
@@ -44,7 +44,8 @@ async function handleCategory(req, res) {
     createdAt: moment(meta.createdTime).fromNow(),
     editLink: meta.mimeType === 'text/html' ? meta.folder.webViewLink : meta.webViewLink,
     id,
-    template: stringTemplate
+    template: stringTemplate,
+    duplicateNames: duplicates ? duplicates.join(', ') : ''
```

Comment:
What do you think about passing the `duplicates` directly into the render call so that future changes wouldn't touch this file? Then we could perform the `.join()` inside the strings.yaml itself.

Reply:
👍